RFXtrx/lowlevel.py

```diff
@@ -2803,6 +2803,202 @@ def _set_strings(self):
             else:
                 self.cmnd_string = self._UNKNOWN_CMND.format(self.cmnd)
 
+
+###############################################################################
+# DDxxxx class
+###############################################################################
+
+
+class DDxxxx(Packet):
+    """
+    Data class for the DDXxxx packet type
+    """
+
+    PACKET_TYPE = 0x31
+    """
+    Packet type for DDXxxx packets
+    """
+
+    TYPES = {0x00: 'Brel/Dooya DDxxxx'}
+    """
+    Mapping of numeric subtype values to strings, used in type_string
+    """
+
+    CMD_UP = 0x00
+    CMD_DOWN = 0x01
+    CMD_STOP = 0x02
+    CMD_P2 = 0x03
+    CMD_PERCENT = 0x04
+    CMD_ANGLE = 0x05
+    CMD_PERCENT_ANGLE = 0x06
+    CMD_HOLD_UP = 0x07
+    CMD_HOLD_STOP = 0x08
+    CMD_HOLD_UP_DOWN = 0x09
+    CMD_HOLD_STOP_UP = 0x0A
+    CMD_HOLD_STOP_DOWN = 0x0B
```

Comment:
I dont have any commands above percent angle in my docs. Does your indicate something else?

Reply:
Yes, the version of the docs I got is 9.47. It is on page 62/155.